packages/core/contracts/cross-chain-oracle/interfaces/ChildMessengerInterface.sol

```diff
@@ -4,8 +4,4 @@ pragma solidity ^0.8.0;
 interface ChildMessengerInterface {
     // Should send cross-chain message to Parent messenger contract or revert.
     function sendMessageToParent(bytes memory data) external;
-
-    // Should be called by parent messenger over the canonical bridge. Includes a target as the L2 contract that should
-    // have data passed to it.
-    function processMessageFromCrossChainParent(bytes memory data, address target) external;
```

Comment:
why did you remove this?

Reply:
Because we do not need it in `Polygon` messengers, and I see no harm in adding it to the Optimism adapters